image_width: 848
image_height: 480
camera_name: camera
distortion_model: plumb_bob
camera_matrix:
  rows: 3
  cols: 3
  dt: d
  data:
  - 690.48388671875
  - 0.0
  - 645.510986328125
  - 0.0
  - 690.1282958984375
  - 358.9439697265625
  - 0.0
  - 0.0
  - 1.0
distortion_coefficients:
  rows: 1
  cols: 8
  dt: d
  data:
  - 0.008727164939045906
  - -0.04575484246015549
  - 0.0004927990376017988
  - 0.00025344331515952945
  - 0.030775338411331177
  - 0.0
  - 0.0
  - 0.0
rectification_matrix:
  rows: 3
  cols: 3
  dt: d
  data:
  - 1.0
  - 0.0
  - 0.0
  - 0.0
  - 1.0
  - 0.0
  - 0.0
  - 0.0
  - 1.0
projection_matrix:
  rows: 3
  cols: 4
  dt: d
  data:
  - 690.48388671875
  - 0.0
  - 645.510986328125
  - 0.0
  - 0.0
  - 690.1282958984375
  - 358.9439697265625
  - 0.0
  - 0.0
  - 0.0
  - 1.0
  - 0.0
